wandb/board/ui/src/util/runhelpers.js

```diff
@@ -440,7 +441,13 @@ export class RunFancyName {
 export function defaultViews(run) {
   //TODO: do we need to handle this case?
   if (!run) run = {summaryMetrics: '{}'};
-  const scalars = Object.keys(JSONparseNaN(run.summaryMetrics));
+  const scalars = Object.keys(
+    JSONparseNaN(
+      run.history && run.history.length > 0
+        ? run.history[0]
+        : run.summaryMetrics
```

Comment:
Figured we should use history in case someone is setting summary to something different.